src/interfaces/ISparkConduit.sol

```diff
@@ -152,13 +152,6 @@ interface ISparkConduit is IAllocatorConduit {
             uint256 totalRequestedFunds
```

Comment:
Need to add all the new functions `totalShares`, `totalRequestedShares`, etc.

Reply:
Yeah agreed, made a ticket for it https://linear.app/phoenixlabs/issue/SC-136/override-all-functions